geode/client-server-examples/basic-operations/src/test/java/example/springdata/geode/client/basic/BasicClientTest.java

```diff
@@ -0,0 +1,114 @@
+/*
+ *
+ *  * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ *  * agreements. See the NOTICE file distributed with this work for additional information regarding
+ *  * copyright ownership. The ASF licenses this file to you under the Apache License, Version 2.0 (the
+ *  * "License"); you may not use this file except in compliance with the License. You may obtain a
+ *  * copy of the License at
+ *  *
+ *  * http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software distributed under the License
+ *  * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ *  * or implied. See the License for the specific language governing permissions and limitations under
+ *  * the License.
+ *
+ */
+
+package example.springdata.geode.client.basic;
+
+import example.springdata.geode.client.basic.client.BasicClientApplicationConfig;
+import example.springdata.geode.client.basic.client.CustomerRepository;
+import example.springdata.geode.client.basic.server.Server;
+import org.apache.geode.cache.Region;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.data.gemfire.tests.integration.ForkingClientServerIntegrationTestsSupport;
+import org.springframework.data.gemfire.util.RegionUtils;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import javax.annotation.Resource;
+import java.io.IOException;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.NONE, classes = BasicClientApplicationConfig.class)
+@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_CLASS)
+public class BasicClientTest extends ForkingClientServerIntegrationTestsSupport {
```

Comment:
Please rename all test classes from `…Test` to `…Tests`. Right now, running `mvn test` does not execute the test classes (inclusion pattern is set to `**/*Tests.java`). We also should make sure that tests complete within a reasonable time.

Reply:
Renamed.